exporter/fileexporter/README.md

```diff
@@ -33,7 +35,7 @@ The following settings are optional:
   - localtime : [default: false (use UTC)] whether or not the timestamps in backup files is formatted according to the host's local time.
 
 - `format`[default: json]: define the data format of encoded telemetry data. The setting can be overridden with `proto`.
-
+- `compression`[default:false]:defines whether to compress encoded telemetry data.
```

Comment:
We may eventually want to support other compression algorithms. Instead of using a boolean, why don't we use a string? The default can be an empty string, and this will imply no compression. `"zstd"` can act the same as you currently have for `true`.

Reply:
Thanks for your advice～